src/components/base/dropdown2/NavigationDropdown.js

```diff
@@ -14,12 +14,9 @@ import Text from "@/components/base/text";
  * This component should standardize the accessability concerns in creating a mix between an accordion and a link list.
  */
 
-function LinkDropdown({
-  context,
-  menuItems,
-  uniqueIdButton = "linkmenu",
-  uniqueIdMenu = "menuButton",
-}) {
+function LinkDropdown({ context, menuItems }) {
+  const uniqueIdButton = "linkmenu";
```

Comment:
Hvorfor fjerne dem fra parametre? Det fint det her, men jeg undrer mig bare

Reply:
man har nok ikke to navs på den samme side, derfor tænker jeg, man kan godt hardcode dem.